website/content/en/docs/building-operators/golang/testing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Testing your Operator project
+linkTitle: Testing with EnvTest
+description: Learn how to ensure the quality of your Operator project 
+weight: 70
+---
+
+## Overview
+
+The Operator SDK project recommends controller-runtime's [envtest][envtest] because it has a more active contributor community, it has become more mature than Operator SDK's test framework, and it does not require an actual cluster to run tests, which can be a huge benefit in CI scenarios. 
+
+## Using EnvTest
+
+You will see that `controllers/suite_test.go` is created. This file contains boilerplate for executing integration tests using [envtest][envtest] with [ginkgo](https://onsi.github.io/ginkgo/) and [gomega](https://onsi.github.io/gomega/).
+
+It means that you will implement your controller's tests in go using this stack and it will be called by the `suite_test.go` via go tool such as: 
+
+```shell
+go test controllers/ -v -ginkgo.v
+```
+
+The projects generated by using the SDK tool has a Makefile which will contain the target `tests` that executes this tests by default when, for example, you run `make tests` your `make docker-build IMG=<some-registry>/<project-name>:<tag>`.  
```

Comment:
should be `projects...have` or `project..has`

Reply:
maybe "executes the tests when you `make tests`? I'm not sure what the rest of the sentence but about make docker-build is about, tho.

Reply:
tks, I tried clarifies that. I would like to make clear that the tests are also executed when we run make docker build.